operator/webhooks/pod_mutating_webhook.go

```diff
@@ -62,10 +78,169 @@ func (a *PodMutatingWebhook) InjectDecoder(d *admission.Decoder) error {
 
 func (a *PodMutatingWebhook) isKeptnAnnotated(pod *corev1.Pod) bool {
 	_, gotApplicationAnnotation := pod.Annotations["keptn.sh/application"]
-	_, gotServiceAnnotation := pod.Annotations["keptn.sh/application"]
+	_, gotServiceAnnotation := pod.Annotations["keptn.sh/service"]
```

Comment:
let's introduce a constant for those annotations in a `common` package and use that everywhere those are used